src/Microsoft.ML.Transforms/Text/StopWordsRemoverTransform.cs

```diff
@@ -278,7 +278,7 @@ private static NormStr.Pool[] StopWords
                 if (_langsDictionary == null)
                 {
                     var langsDictionary = Enum.GetValues(typeof(Language)).Cast<Language>()
-                        .ToDictionary(lang => lang.ToString().AsMemory());
+                        .ToDictionary(lang => lang.ToString().AsMemory(), new TextAnalytics.ReadonlyMemoryComparer());
```

Comment:
>new TextAnalytics.ReadonlyMemoryComparer() [](start = 74, length = 42)

Wouldn't it be easier to go other way around? and cast language from column from readonly<char> to string?

Reply:
It would be easier for me :-), but I think it would be more expensive to create a new string for every row.

---
In reply to: [234047532](https://github.com/dotnet/machinelearning/pull/1641#discussion_r234047532) [](ancestors = 234047532)